src/configprovider-cppdbg.ts

```diff
@@ -10,6 +10,8 @@ export enum MIModes {
 }
 
 export class DebugConfigurationProviderCppdbg implements vscode.DebugConfigurationProvider {
+  static readonly type = "cppdbg";
```

Comment:
Rename it to `name`. Thanks for this commit.

Reply:
Static property 'name' conflicts with built-in property 'Function.name' of constructor function 'DebugConfigurationProviderCppdbg'.

Reply:
type is fine then!